src/panels/lovelace/cards/hui-alarm-panel-card.ts

```diff
@@ -353,8 +353,7 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
       }
 
       .actions {
-        margin: 0 8px;
-        padding-top: 20px;
+        margin: 0;
```

Comment:
On smaller screens the actions are now touching the badge?

Reply:
Even on the smallest simulated screen, there is still space between them
![image](https://user-images.githubusercontent.com/114137/99957636-503e6280-2d88-11eb-9975-85194aa73ebc.png)